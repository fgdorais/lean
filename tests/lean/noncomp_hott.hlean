noncomputable definition foo := 10
