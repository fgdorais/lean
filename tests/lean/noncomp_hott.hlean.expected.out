noncomp_hott.hlean:1:0: error: invalid 'noncomputable' declarations, it can only be used in the standard library
